tests/jvm/src/test/resources/lowlevel.expect

```diff
@@ -372,7 +372,7 @@ flags.p.package.z(Int).(pp) => param pp: Int
   Int => scala.Int#
 local0 => val local xs2: Nothing
   Nothing => scala.Nothing#
-local1 => abstract local t: >: Nothing <: Any
+local1 => abstract val local t: >: Nothing <: Any
```

Comment:
This was the only noticeable change from my refactoring of kinds/properties. I don't understand why it changed since it's seemingly a direct translation of the previous implementation


https://github.com/scalameta/scalameta/blob/110147dfa610b7859623ff2aad7f5ee3abbc60b3/semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/DenotationOps.scala#L97-L99
```scala
// before
        if ((kindFlags & (mf.LOCAL | mf.FIELD)) != 0) {
          if (gsym.isMutable) flags |= mf.VAR
          else flags |= mf.VAL
// after
        if (kind.isLocal || kind.isField) {
          if (gsym.isMutable) flip(p.VAR)
          else flip(p.VAL)
```

Either way, I've added a `gsym.isVal` guard before `flip(p.VAL)`. The long-term fix is https://github.com/scalameta/scalameta/issues/1503

Reply:
This guard reverts removes the diff from this PR.